### ISSUE TITLE
Authentication not proceeding after introduction of JWT

### ISSUE THREAD

**Issue by kushthedude** [layer5io/meshery#891]
O-Auth page doesn't move ahead and the invalid token is generated by the JWT implemented.

For more details see the post-merge conversation on #757.

Error Logs :
```sh
ERRO[0010] error in getting the token, http: named cookie not present
ERRO[0010] Error: unable to get session: http: named cookie not present
DEBU[0021] token : eyJhY2Nlc3NfdG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0ltdHBaQ0k2SW5CMVlteHBZenBsT0dWbU5ERmpNeTFpWldWbUxUUmlZakV0T0dVNE1DMHpOakExTVRZeU4yTTJNakVpTENKMGVYQWlPaUpLVjFRaWZRLmV5SmhkV1FpT2x0ZExDSmpiR2xsYm5SZmFXUWlPaUp0WlhOb1pYSjVMV05zYjNWa0lpd2laWGh3SWpveE5UZzNOamMxTnpRMUxDSmxlSFFpT250OUxDSnBZWFFpT2pFMU9EYzJOekl4TkRRc0ltbHpjeUk2SW1oMGRIQnpPaTh2YldWemFHVnllUzVzWVhsbGNqVXVhVzh2YUhsa2NtRXZJaXdpYW5ScElqb2lNREUxWmpZME1tUXRaakpsTnkwME1EQmhMVGswTVRjdFlUQXhaRGxrTVRreE1UQXlJaXdpYm1KbUlqb3hOVGczTmpjeU1UUTBMQ0p6WTNBaU9sc2liM0JsYm1sa0lpd2liMlptYkdsdVpTSmRMQ0p6ZFdJaU9pSmhNMVo2WVVoU2IxcFhVakZhUjFVOU9sb3liREJoU0ZacEluMC5QdUZRVVZsaTAtSGFVbTdjWm5ZQUNKOFV6ZzlSZnlVX3BkMXl4TllNZmJQY2NTQXFSblR0VFVpdUN6UFFxVm9EODJyanJHN2Y2RXhnQ21mZlI5S3JqSWRMaG5vRnprNmR2ZTFONWhyZGpUdjN4aUk2YlZhR3o2Vk16VGM1bmdxQzY3ZzZkb3JrV3ozaU41TmdtWHVxUkhmUmdMOHVwWW1ub1RVaTVZTmpJZkJzM1lJbUJaU2xJd2haVnhHUHFLWGJBUG9LbzlRNHRBRV8xTWdEakU2d3d4YkdFbFBWMGI3bmcxaktXbjYxRGVZUFFfU3BOVk4tUzZoZkJhUUl6VUdydmFrUE14MjY3ei00T1hzaFVQcWdNVzF5aVVidTB1RHBqOHh4cS1taEE3aTFzdGdQV0JBSmJRa0hXSF9KVEl4SFYyb1JLS1FBbGNIWVRGaW16RUVyT2xjY1BxM3FMWVlYY2hTZmVvb1ZjaVlsaDRONXRGWHBCUFBvQTZqeFhxZUVWVDU2X1IyVTJ4cElXZzY2Mndid1JGT2Nhd1RSQXY1UzRCQ3kwdExpUWVFRzgzaXpMWFBMX2VfTXhQdVE5emF4LWl6M1lhU3Ztb0t2eUluLVU1RUtLdmc5ckhVSUR0SnFjMFF4X25Sa2RzeEhJUEpvUHV6QU5FTDFkUV9mX0NWU1RCcS1ySS1BcEdQWmFUUHRCeFlSMEV3cFphalVPNnR6UF85UnpScnQyRDBZQkkyYXlPUV9Zb3BpZWdyd0lGYnVYRm1DV2tkZkxLYWlOVEpfN2ViczRuZ25NV1dPanhSTDc3UWlrd2swU2UxcWNkbTEwSDVfaUVFVjE0NG9FRVQ2SmJ1YkFJM2RhNWR1SkMxSXdMZm1nUEZ1ODVIWUQtbktZS1RGYmFjdmZzayIsInRva2VuX3R5cGUiOiJiZWFyZXIiLCJyZWZyZXNoX3Rva2VuIjoiOXV0YUdLdzFYckJFMkFrNFNZRG51MnBBQTVvSVFPQjVPR2RQOWxseS04Yy5WZHNBWUdHSWFRNnpvSmlKNmpmdl9MMkxGQnpib3M5VVJLX2RqQTVKSW5VIiwiZXhwaXJ5IjoiMjAyMC0wNC0yM1QyMTowMjoyNC4xMTg5MDYxNDJaIn0
INFO[0023] Error: parsing token, trying refreshing the token
ERRO[0023] Error refreshing token, Token is not valid yet
DEBU[0031] token : eyJhY2Nlc3NfdG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0ltdHBaQ0k2SW5CMVlteHBZenBsT0dWbU5ERmpNeTFpWldWbUxUUmlZakV0T0dVNE1DMHpOakExTVRZeU4yTTJNakVpTENKMGVYQWlPaUpLVjFRaWZRLmV5SmhkV1FpT2x0ZExDSmpiR2xsYm5SZmFXUWlPaUp0WlhOb1pYSjVMV05zYjNWa0lpd2laWGh3SWpveE5UZzNOamMxTnpVMUxDSmxlSFFpT250OUxDSnBZWFFpT2pFMU9EYzJOekl4TlRVc0ltbHpjeUk2SW1oMGRIQnpPaTh2YldWemFHVnllUzVzWVhsbGNqVXVhVzh2YUhsa2NtRXZJaXdpYW5ScElqb2lZVEZrWW1SbE9HTXROR1k1WXkwME0ySmtMVGt4TmpBdE5qZzNNVGMzTVRZek9EZzNJaXdpYm1KbUlqb3hOVGczTmpjeU1UVTFMQ0p6WTNBaU9sc2liM0JsYm1sa0lpd2liMlptYkdsdVpTSmRMQ0p6ZFdJaU9pSmhNMVo2WVVoU2IxcFhVakZhUjFVOU9sb3liREJoU0ZacEluMC5KbmtKNEJKYUtiVlJFRjREd0ZFdnFFT3ZKOUFaVEVmVDNmV0VVUnhSZHZsbjNnaFdjdEJxRzFOSEdoLUN5bXhZM1d0Mzh0R2hRbnZEMEpIalcwdHlvWnYweFUwYVpEYm9USGVpQzNiUGpXWkJWZDMtbk1CcjhPUE1kRC1rcElaa1R3V1I3dzJ1SGJMWmVhNHdrSlFNdU9PSHpkWmh2cVd5MWdOTG8tSXZoOWxVSUF0SXRCR0c3cXZ2VG5YRVBIcWtLeldEVVdZZFZQWGRjcUxrZk1vbXdXTFNxcGp0eWxCcldrTUtWcFluZE5kMTB5S2VFVkdBU0dWaE1IeVNzMnpKUkxsTWRUc3Q3OG1QSHZhY3RHbFlHWUVMNmtVcktkWmJxcWFRTFJqRHUxRWtibDB0aVUxdy1pdkZTM21vVW1Na3I3bWRrRWtTRHh5NEpxVTQ1UzJhZ0oxZ2Y3QVVnMUdudE95bmF6UXpUTFVMbzMxMThZTHpHNW14cHVqRmJ2Z3h3SjhhRWI5V01tRDRsQnlMMHktaEdzOVEzcjNsYlJDRXFUVFNQZ1dWOC1Ra0RQR3lUNHp3UkpHVU5sU1ZyR3ZPcWd6TVV5YUp5aW9HZGNnbzVTUTN2dlpUdEp0OFA4NmNkZ2UxWURPbFYxV0xhRFpkdFlPcEJjS1Z4TGwyVkdpZ3lxY2J5Z3NuSUI5Z2hJZDJoUUVzQXQtRC1nUDRHcG01eG1CcVU1WGpDdXdRSTRGN0xJZUdyU1MxNk0tU3ZKMGxGcTdYY2xtcjZKOHA1WU1fSGlQT0EzUjUwdGtfX2ZzSGMxM2xhMF9KRlpPaGk5ZlhWUkFfeFViREdaY25XRkZETFhIdzdwaWN0M1hTVnl4eTUzSzR4aEtEdFgyYVlldE9VLU5QT2Z6ZzVsQSIsInRva2VuX3R5cGUiOiJiZWFyZXIiLCJyZWZyZXNoX3Rva2VuIjoiZy1LdFBuWXlSQ01MeXVFVkxIWmpqNXFVZ1o2elpzVzFFNGJlbnNIbHlZVS54X3NYMlEyYjdVNXo0blBNbEJfWDROOWxrT2h3QzdoMkRPTzdQWWNweTlnIiwiZXhwaXJ5IjoiMjAyMC0wNC0yM1QyMTowMjozNC41MzA0OTc4MjdaIn0
INFO[0031] Error: parsing token, trying refreshing the token
ERRO[0031] Error refreshing token, Token is not valid yet
```

**Comment by kanishkarj** [comment]
The token produced is valid (I checked), for some reason meshery is not able to validate them (on your or a few systems)... I'll let you know once I make more progress

**Comment by kushthedude** [comment]
> The token produced is valid (I checked), for some reason meshery is not able to validate them (on your or a few systems)...

I just wrote what logs told me. 

> I'll let you know once I make more progress

Sure

**Comment by aayushdutt** [comment]
This issue is most probably fixed after #892

**Comment by kushthedude** [comment]
Closed on #892
